chaco/ticks.py

```diff
@@ -205,7 +209,14 @@ def auto_ticks ( data_low, data_high, bound_low, bound_high, tick_interval,
 
     if upper > end:
         end += tick_interval
-    ticks = arange( start, end + (tick_interval / 2.0), tick_interval )
+
+    if isinf(start) or isinf(end + (tick_interval / 2.0)):
```

Comment:
Hmmm... I agree with you. Regardless of `tick_interval`, `start` will be a `float` because that's what `floor` will always return. Not sure what that other guy was thinking.